FirebaseAuth/Sources/Backend/RPC/FIRGetOOBConfirmationCodeRequest.m

```diff
@@ -280,10 +295,18 @@ - (nullable id)unencodedHTTPRequestBodyWithError:(NSError *_Nullable *_Nullable)
   if (_handleCodeInApp) {
     body[kCanHandleCodeInAppKey] = @YES;
   }
-
```

Comment:
super-nit: restore newline and add newlines between if-blocks to be consistent with the rest of the code here?

Reply:
Blank lines removed to be consistent with other files.